SearchBar.js

```diff
@@ -1,15 +1,11 @@
-var NativeModules, PropTypes, RNSearchBar, React, SearchBar;
+import { Component, PropTypes } from 'react';
+import React, { NativeModules, requireNativeComponent } from 'react-native';
```

Comment:
I should maybe call the variable import ReactNative from 'react-native';

But I need import from 'react-native' to use ReactNative.findNodeHandle.

BTW I tested it on RN 0.25 and it works without warnings.